package.json

```diff
@@ -47,7 +48,7 @@
     "validate.io-nonnegative-integer": "^1.0.0",
     "validate.io-number-primitive": "^1.0.0",
     "validate.io-object": "^1.0.4",
-    "validate.io-positive-primitive": "^1.0.0",
+    "validate.io-positive": "^1.0.0",
```

Comment:
```suggestion
    "validate.io-positive-primitive": "^1.0.0",
```

Reply:
This should not be changed.

Reply:
Why not? This library uses `validate.io-positive`, not `validate.io-positive-primitive` (https://github.com/distributions-io/beta-pdf/blob/master/lib/validate.js#L6). Even the test fails before the changes in this PR.

Reply:
Ah...then the `require` path in `validate.js` should be changed, instead.